src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +52,37 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+      //ToDo(kristenmason@): display map
+      this.createButtons_();
+    })
```

Comment:
Consider adding a catch section here for if the player does not allow you to pause.

Reply:
good idea!